package.json

```diff
@@ -59,9 +59,9 @@
     "@typescript-eslint/eslint-plugin": "^5.12.1",
     "@typescript-eslint/parser": "^5.12.1",
     "archiver": "^5.3.0",
```

Comment:
Is this still needed? I think it's used by cdk to create the asset archives but just checking.

Reply:
indeed might not be needed. will merge that one to let @ijemmy work and give it a try through another PR